extensions-core/datasketches/src/test/java/org/apache/druid/query/aggregation/datasketches/hll/HllSketchMergeAggregatorFactoryTest.java

```diff
@@ -66,6 +76,11 @@ public void setUp()
         SHOULD_FINALIZE,
         !ROUND
     );
+
+    EasyMock.expect(metricFactory.getColumnCapabilities(EasyMock.anyString())).andReturn(capabilities).anyTimes();
+    EasyMock.expect(vectorFactory.getColumnCapabilities(EasyMock.anyString())).andReturn(capabilities).anyTimes();
+    EasyMock.expect(capabilities.toColumnType()).andReturn(ColumnType.NESTED_DATA).anyTimes();
+    EasyMock.replay(metricFactory, vectorFactory, capabilities);
```

Comment:
Let's not mock everything. Please reuse existing classes instead of mocking the selectors